WORKSPACE

```diff
@@ -1,5 +1,22 @@
 workspace(name = "org_pubref_rules_protobuf")
 
+
+android_sdk_repository(
+    name = "androidsdk",
+    # Replace with path to Android SDK on your system
+    path = "/Users/pcj/Library/Android/sdk",
```

Comment:
As of bazel 0.4.5 you (reliably) expect the repository rule to read the sdk location from the `ANDROID_HOME` variable.